src/schema/v2/viewingRoom.ts

```diff
@@ -0,0 +1,40 @@
+import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
+import cached from "./fields/cached"
+import { InternalIDFields } from "./object_identification"
+import { ResolverContext } from "types/graphql"
+
+export const ViewingRoomType = new GraphQLObjectType<any, ResolverContext>({
```

Comment:
yeah I _feel_ like we shouldn't need this since we define this type in Gravity's GraphQL fields and stitch it in...would def like to learn more about why this would/would not be necessary because I'm not sure if we've set up this Vortex flow with a Gravity GQL type before! E.g. shows and artists are queried via REST endpoints w/ Gravity loaders, so we might actually be able to do less (or at least different) stuff with ViewingRooms since they exist as GQL fields.

This might also relate to the authentication you discussed above - the Gravity REST endpoint I _think_ is used for partners, while the GQL fields are what we use for displaying in Eigen/Force. We might not need to use loaders at all, and therefore not have to worry about auth, if we use the `ViewingRoom` field provided by Gravity GQL.

Reply:
Addressed offline! 

Matt is exactly right here - Viewing Rooms are accessed via Gravity’s Graphql interface where the other objects in this pattern were accessed by gravity data loaders. So I was barking up (🐕 ) the wrong tree in these early comments